composer.json

```diff
@@ -11,7 +11,7 @@
         {"name": "Michael Simonson", "email": "contact@mikesimonson.com" }
     ],
     "require": {
-        "php": ">=5.4.0",
+        "php": "^5.5|^7.0",
```

Comment:
Silly question, but will this works with php 5.6? According to [composer docs](https://getcomposer.org/doc/articles/versions.md#caret) `^5.5` === `>=5.5.0 <5.6.0`.

Does `~5.5` will be better?